image-tag

```diff
@@ -8,5 +8,10 @@ WORKING_SUFFIX=$(if ! git diff --exit-code --quiet HEAD >&2; \
                  then echo "-WIP"; \
                  else echo ""; \
                  fi)
-BRANCH_PREFIX=$(git rev-parse --abbrev-ref HEAD)
-echo "${BRANCH_PREFIX//\//-}-$(git rev-parse --short HEAD)$WORKING_SUFFIX"
```

Comment:
For consistency, should `${CIRCLE_BRANCH//\//-}` be changed in `circle.yml` as well?